client/cherami/consumer.go

```diff
@@ -213,22 +222,27 @@ func (c *consumerImpl) reconfigureConsumer() {
 	c.lk.Lock()
 	defer c.lk.Unlock()
 
+	var err error
+
 	select {
 	case <-c.closingCh:
 		c.logger.Info("Consumer is closing.  Ignore reconfiguration.")
 	default:
 		var conn *outputHostConnection
 
-		consumerOptions, err := c.client.ReadConsumerGroupHosts(c.path, c.consumerGroupName)
-		if err != nil {
-			c.logger.Warnf("Error resolving output hosts: %v", err)
-			if _, ok := err.(*cherami.EntityNotExistsError); ok {
-				// ConsumerGroup is deleted. Continue with reconfigure and close all connections
-				consumerOptions = &cherami.ReadConsumerGroupHostsResult_{}
-			} else {
-				// This is a potentially a transient error.
-				// Retry on next reconfigure
-				return
+		consumerOptions := &cherami.ReadConsumerGroupHostsResult_{}
+		if atomic.LoadUint32(&c.paused) == 0 {
+			consumerOptions, err = c.client.ReadConsumerGroupHosts(c.path, c.consumerGroupName)
+			if err != nil {
```

Comment:
Just curious, how we pause consuming here when c.paused is zero? It seems it will continue executing when error is nil.